lib/utils.ts

```diff
@@ -8,4 +8,66 @@ function isNavigatorAvailable(): boolean {
     return typeof navigator !== 'undefined';
 }
 
-export {isWindowAvailable, isNavigatorAvailable};
+const htmlEscapes = {
+    '&': '&amp;',
+    '<': '&lt;',
+    '>': '&gt;',
+    '"': '&quot;',
+    "'": '&#x27;',
+    '`': '&#x60;',
+};
+const reUnescapedHtml = /[&<>"'`]/g;
+const reHasUnescapedHtml = RegExp(reUnescapedHtml.source);
+
+/**
```

Comment:
To every function copied from underscore/lodash, let's add an annotation/link to the source